packages/main/src/StepInput.ts

```diff
@@ -512,12 +546,22 @@ class StepInput extends UI5Element implements IFormElement {
 		}
 	}
 
+	_isPrecisionValid() {
```

Comment:
Can we think of a better name? I think it doesn't give enough information. What I understand is: Is the precision valid, like is the property called precision valid.

Reply:
Does `_isValuePrecisionValid` sound better to you ?

Reply:
Maybe _isValueWithCorrectPrecision ? Kind of longer, but more descriptive.